commands/verify/verify.go

```diff
@@ -3,93 +3,90 @@ package verify
 import (
 	"context"
 	"fmt"
-	"os"
 	"path/filepath"
-	
-	"github.com/instrumenta/conftest/policy"
+
 	"github.com/instrumenta/conftest/commands/test"
-	"github.com/containerd/containerd/log"
+	"github.com/instrumenta/conftest/policy"
 	"github.com/open-policy-agent/opa/tester"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
-func NewVerifyCommand(getOutputManager func() test.OutputManager) *cobra.Command {
-	ctx := context.Background()
-	cmd := &cobra.Command{
+func NewVerifyCommand(ctx context.Context) *cobra.Command {
+	cmd := cobra.Command{
 		Use:   "verify",
 		Short: "Verify Rego unit tests",
-		PreRun: func(cmd *cobra.Command, args []string) {
+		PreRunE: func(cmd *cobra.Command, args []string) error {
 			err := viper.BindPFlag("output", cmd.Flags().Lookup("output"))
 			if err != nil {
-				log.G(ctx).Fatal("Failed to bind argument:", err)
+				return fmt.Errorf("bind output: %w", err)
 			}
-		},
-		Run: func(cmd *cobra.Command, args []string) {
-			out := getOutputManager()
 
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I like the use of RunE here, so we can get a return value instead of the log.Fatals that were there
👍